CMakeLists.txt

```diff
@@ -1,28 +1,28 @@
 cmake_minimum_required (VERSION 2.6)
 project (graywolf)
 
-find_package(PkgConfig)
+INCLUDE(GNUInstallDirs)
 INCLUDE(CheckIncludeFiles)
 
+FIND_PACKAGE(BISON REQUIRED)
+FIND_PACKAGE(FLEX REQUIRED)
+FIND_PACKAGE(GSL REQUIRED)
+FIND_PACKAGE(PkgConfig)
 
 pkg_check_modules(GSL gsl)
 
 if (NOT GSL_FOUND)
   MESSAGE(FATAL_ERROR "The development files for the GNU Scientific Library (libgsl) are required to build graywolf.")
 endif()
 
-# Include RPATH in build so that ldconfig is not necessary after install
-SET(CMAKE_SKIP_BUILD_RPATH FALSE)
```

Comment:
So the RPATH was added on Tim's request. That way he said he avoids many support requests. Common distro packaging scripts strip out the RPATH anyway when building packages, so I cannot see that these lines harm.